### PR TITLE
chore: update jest and jest-config

### PR DESCRIPTION
**Pull request by sdobbert**
### Summary
This merge request updates jest, the jest-config file and the report paths for SonarQube. This is neccessairy to report the jest test results correctly so that they can be scanned and displayed by sonarQube.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-    PROJECT_VERSION: '42.0.0',
+    PROJECT_VERSION: JSON.stringify(require('./package.json').version),
     KEYCLOAK_HOST: 'localhost',
     KEYCLOAK_REALM: 'SHOGun',
     KEYCLOAK_CLIENT_ID: 'shogun-client'
   },
   transform: {
-    '^.+\\.jsx?$': '<rootDir>/node_modules/babel-jest',
-    '^.+\\.tsx?$': '<rootDir>/node_modules/babel-jest'
+    '^.+\\.jsx?$': 'babel-jest',
```
Comment (FritzHoing) [inline jest.config.js:10]: Out of curiosity, why does this need to be adjusted?

**Comment by FritzHoing** [comment]
Looks good to me! Did you test the sonarqube-pipeline already?

File: jest.config.js (hunk 1 above)
Comment (sdobbert) [reply jest.config.js:10]: It is not strictly neccessairy, just more simple/clean.

**Comment by sdobbert** [comment]
Thank you! Yes, the sonarqube-pipeline was tested already and does report the coverage correctly now:
image

**Review by FritzHoing — APPROVED** [review]

**Pull request merged**